examples/simple.ts

```diff
@@ -1,11 +1,28 @@
+import { OpenAI as OriginalOpenAI } from "openai";
+import { zodResponseFormat } from "openai/helpers/zod";
+import { z } from "zod";
 import { objectTemplate } from "../src";
 import { OpenAI } from "../src/client";
 
+const Step = z.object({
```

Comment:
This is using the example from OpenAI to see if i could get refusals to show up. I was planning on keeping here for future us since it doesnt fail any tests.

Could also allow us to test out `parse` when we implement it because right now `parse` is broken when using Lirbetto's sdk.